tests/composition_velocity_zero.prm

```diff
@@ -0,0 +1,25 @@
+# This is a test for the composition velocity statistics
+# postprocessor, specifically, if it also works when a
+# field is zero.
+
+include $ASPECT_SOURCE_DIR/tests/composition_active.prm
+
+set End time = 0.2
+
+subsection Compositional fields
+  set Number of fields = 3
+end
+
+subsection Initial composition model
+  subsection Function
+    set Function expression = if(y<0.2, 1, 0) ; if(y>0.8, 1, 0); 0.0
+  end
+end
+
+subsection Postprocess
+  set List of postprocessors = visualization, temperature statistics, composition statistics, composition velocity statistics
+
+  subsection Composition velocity statistics
+    set Names of selected compositional fields =
```

Comment:
Would it be worth adding a test that ensures the changes work when this parameter is not blank (or maybe a test already exists for this)?